Modules/IO/TIFF/src/itkTIFFImageIO.cxx

```diff
@@ -387,8 +387,8 @@ TIFFImageIO::ReadImageInformation()
   m_Origin[0] = 0.0;
   m_Origin[1] = 0.0;
 
-  int width = m_InternalImage->m_Width;
-  int height = m_InternalImage->m_Height;
+  size_t width = m_InternalImage->m_Width;
+  size_t height = m_InternalImage->m_Height;
```

Comment:
Again, I agree with you that `width` and `height` should not be signed int, but I'm not convinced about `size_t either, here. in this case I would not even declare them. `m_InternalImage->m_Width` and `m_Height` could be assigned directly to m_Dimensions[i], right?

Reply:
yep